### DISCUSSION TITLE
with-sentry example: NEXT_PUBLIC_SENTRY_SERVER_ROOT_DIR?

### DISCUSSION THREAD

**Discussion by osdiab** [vercel/next.js#19092] [Help]
Hi all, I'm new to NextJS and see the environment variable `NEXT_PUBLIC_SENTRY_SERVER_ROOT_DIR` being used in the `with-sentry` example, but I can't find any docs about it. Where does that come from? Thanks!

**Comment by jvanmaaren** [comment]
Same question here! 

The example mentions _"In the Vercel environment, this is /var/task/."_

But what would this be for a basic Next project?

**Reply by jvanmaaren** [reply]
So, in Dockfile I'm using `WORKDIR /code/`, so this variable should probably be `/code/`

**Reply by antkougblenou** [reply]
have you ever found the answer ?
